src/main/java/bio/terra/app/usermetrics/BardClient.java

```diff
@@ -41,13 +40,12 @@ public class BardClient {
   private static final String SYNC_PATH = "/api/syncProfile";
 
   @Autowired
-  public BardClient(UserMetricsConfiguration metricsConfig) {
-    this.restTemplate = new RestTemplate();
-    restTemplate.setRequestFactory(new HttpComponentsClientHttpRequestFactory());
```

Comment:
Flagging for reviewers: is there a reason that we weren't using the common `RestTemplate` bean before?

Reply:
Nope! I think this might have predated having the shared bean.  Good catch!